src/components/Layout/Layout.tsx

```diff
@@ -33,13 +37,28 @@ export class Layout extends React.Component<LayoutProps, undefined> {
                         {getLinksWithProps()}
                     </SideBar>
                     <SoundSwitch/>
-                    <SwitchControl>
-                        <TransitionSwitch className="translate-container" classNames="translateY">
-                            {getRoutesWithProps()}
-                        </TransitionSwitch>
-                    </SwitchControl>
+                    <SmartBreakpoint match="min-width: 1440px">
+                        <SwitchControl>
+                            <TransitionSwitch className="translate-container" classNames="translateY">
+                                {getRoutesWithProps()}
+                            </TransitionSwitch>
+                        </SwitchControl>
+                    </SmartBreakpoint>
+                    <SmartBreakpoint match="max-width: 1439px">
+                        <this.wholePageList/>
+                    </SmartBreakpoint>
                 </div>
             </Router>
         );
     }
+
+    protected wholePageList(): JSX.Element {
+        const content = routeProps.map((prop) => <prop.component key={prop.path}/>);
```

Comment:
Isn't `const content` redundant?

Reply:
Also, method `wholePageList()` is redundant too, as I see. Move it into `render()`